filters/include/pcl/filters/impl/crop_hull.hpp

```diff
@@ -113,16 +102,44 @@ pcl::CropHull<PointT>::getHullCloudRange ()
       if (pt[i] > cloud_max[i]) cloud_max[i] = pt[i];
     }
   }
-  
+
   return (cloud_max - cloud_min);
 }
 
 //////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 template<typename PointT> template<unsigned PlaneDim1, unsigned PlaneDim2> void 
 pcl::CropHull<PointT>::applyFilter2D (PointCloud &output)
 {
+  std::vector<int> indices;
```

Comment:
Shall we mark the `PointCloud` versions as deprecated? They are private and unused

Reply:
deleted `CropHull::applyFilter[23]D(PointCloud)`, also deleted `CropHull:applyFilter(PointCloud)`, cause `FilterIndices` already have got implementation

deletion `CropHull::applyFilter` fixed bug: `FilterIndices` funtionality for keepOrganized cloud was missed in original implementation